walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -118,23 +211,30 @@ Look at the right side of your playground screen. Do you see the following code
 
 [type=verificationFail]
 ****
-Check the logs of the *ionic-showcase-server* pod.
+Check the logs of the `ionic-showcase-server` pod.
 
 It should include the string `+connected to messaging service+`.
-{standard-fail-text}
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
 ****
 
 [time=5]
-== Accessing the client app in a browser
+== Running and verifying your DataSync server
+
+The {data-sync-starter} provides:
+
+  - offline operation support
+  - out of the box live updates
+  - conflict resolution
 
-The {data-sync-showcase-app} demonstrates the key capabilities provided by the {data-sync-name} framework.
-It can be run either as a hybrid mobile application, or a Progressive Web App (PWA).
+In this guide we are going to explore capabilities of the datasync by using
```

Comment:
Change to  :   In this guide we are going to explore the capabilities of datasync by using